inc/orderinjection.class.php

```diff
@@ -32,15 +32,21 @@
 }
 
 class PluginOrderOrderInjection extends PluginOrderOrder implements PluginDatainjectionInjectionInterface {
-   public function __construct() {
+   function __construct() {
       $this->table = getTableForItemType(get_parent_class($this));
    }
 
-   public function isPrimaryType() {
+    static function getTable() {
+     
+       $parenttype = get_parent_class();
```

Comment:
I tried using parrent::getTable() and datainjection fails with same symptom as if function was not declared at all. If reverting to $parenttype = get_parent_class(); notation, it works properly.